common/autoconf/basics_windows.m4

```diff
@@ -23,6 +23,25 @@
 # questions.
 #
 
+# ===========================================================================
+# (c) Copyright IBM Corp. 2018, 2018 All Rights Reserved
+# ===========================================================================
```

Comment:
Our convention is to insert just these three lines before the existing copyright; see common/autoconf/configure for an example.